stock_getter/backend_stockgetter/backend_stockgetter/urls.py

```diff
@@ -0,0 +1,21 @@
+"""backend_stockgetter URL Configuration
+
+The `urlpatterns` list routes URLs to views. For more information please see:
+    https://docs.djangoproject.com/en/4.0/topics/http/urls/
+Examples:
+Function views
+    1. Add an import:  from my_app import views
+    2. Add a URL to urlpatterns:  path('', views.home, name='home')
+Class-based views
+    1. Add an import:  from other_app.views import Home
+    2. Add a URL to urlpatterns:  path('', Home.as_view(), name='home')
+Including another URLconf
+    1. Import the include() function: from django.urls import include, path
+    2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
+"""
+from django.contrib import admin
+from django.urls import path
```

Comment:
Is this used somehow here?

Reply:
admin class is used to declare the path for the admin login url - see 'admin.site.urls'
path is not used currently, but is always imported in the beginning of the Django urls.py file as it is used to create every other path redirection, other than the admin one

Reply:
make sense